app/gui2/src/components/ComponentBrowser/input.ts

```diff
@@ -183,7 +185,7 @@ export function useComponentBrowserInput(
     const segments = operandsAsIdents.map((ident) => ident.repr())
     const rawQn = segments.join('.')
     const qn = tryQualifiedName(rawQn)
-    return qn.ok ? qn.value : null
+    return qn.ok ? normalizeQualifiedName(qn.value) : null
```

Comment:
`normalizeQualifiedName` expects _full_ qualified name, but we don't know if user wrote full qualified name. 

I would skip normalization here. We don't expect user should use `Main` inside any path.